qlib/TableMapper.qm

```diff
@@ -740,12 +771,13 @@ on_error table_mapper.rollback();
 
             hash h;
             foreach hash row in (iter) {
+                # FIXME --PQ 21-Mar-2017
```

Comment:
I don't understand this comment; if you mean that the constant data appears in the output as single values; this is documented behavior of the output value for bulk inserts